tests/test_setup_command.py

```diff
@@ -91,6 +91,7 @@ def test_everything(project, runner) -> None:
 maintainer = David Zaslavsky, Stuart Longland
 maintainer_email = diazona@ellipsix.net, me@vk4msl.com
 description = A dummy project with a sophisticated setup.cfg file
+license = MIT
```

Comment:
Yeah, I used `MIT` in the example config here since that matches what's in the classifiers below in the same dummy project `setup.cfg`.

The dual-purpose of this field is confusing to say the least.

Reply:
Yeah... The thing is, the classifiers have to come from a fixed list, so we have to use a real license there (if there's going to be a license classifier at all), but this has no such restriction.

Still, it could be something like "MIT license could go here" as far as I'm concerned.